client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -194,6 +224,39 @@ const PlanOnlyThankYou = ( {
 		},
 	} );
 
+	let upsellProps;
+	if ( promotePlanData ) {
+		upsellProps = {
+			title: translate( 'Advanced tools' ),
+			description: (
+				<>
+					{ translate(
+						'Get the %(planName)s plan for access to WordPress plugins, STFP setup, database credentials, and more, propelling your site to new heights. Upgrade risk-free with our %(refundPeriodDays)s-day money-back guarantee, and pay only the difference ({{s}}%(fullPrice)s{{/s}} %(discountPrice)s).',
+						{
+							comment: 'Upsell for WP Business plan on checkout thank you page',
+							args: promotePlanData,
+							components: { s: <s /> },
```

Comment:
Is that a new HTML tag that I'm not aware of 🙂? All jokes aside, I would use something more explicit so translators know that this refers to bold:
```suggestion
							components: { strong: <strong /> },
```

Reply:
`s` is not shortcut for `strong`, is a tag for `text-decoration: line-through;`

There is one more tag `del` which do the same, but here is case about semantic:

The <del> element represents a removal from the document.
The <s> represents contents that are no longer accurate or no longer relevant.

Reply:
Ohhh. Thanks for the clarification!